packages/stripe_js/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.4.1
+**Features**
+- add method for deferred payment
```

Comment:
we need to port this to the new jsinterop plugin. I am about to merge this soon.